### DISCUSSION TITLE
Is there a way to stitch 2 videos (right+left synched screens) into one file?

### DISCUSSION THREAD

**Discussion by gsaccani** [CommandPost/FCPCafe#319] [General Discussions]
I have content to play across 2 screens that need sync when playing. I'd like to stitch the 2 videos together next to each other, into one file. So I don't have any issue with desynchronisation over time.... Every article I read about stitching is adding videos in sequence one after the other. What I'm looking for is a stitch "next to each other"! Thanks for your help

**Comment by randomeizer** [comment]
I think what you're saying is that you want one video that plays across two screens?

Assuming you can find some software that lets you make a single video spread over multiple screens (which may actually be the tricky part), then the FCP part of that would be creating a custom video resolution that's essentially two screens wide and one high.

For example, if both your screens are 1080P, you need to specify a custom resolution of `3840x1080` when creating your Project in FCP. If the screens are UHD/4K, the resolution would be `7680x2160`.

**Comment by gsaccani** [comment]
Yes I do have a post processing system that can split the one video to 2 screens. I do understand the need to setup a specific format for my FCP project. Can I just drag 2 different 1080P videos into fcp and lay them out to be playing in the same project left & right, side by side? I could not find any help on how to achieve this. Thanks, GillesSent from my iPhone

**Reply by randomeizer** [reply]
Basically, you would put one on the Primary Storyline and the other directly above it as a connected clip. Initially, they will be directly on top of each other, and you will just see the top one, like so:

![SCR-20240105-dwb](https://github.com/CommandPost/FCPCafe/assets/3589315/ef314fc4-4cb6-4463-8e88-ac2ff517e005)

The fix is to shift one to the left, the other to the right. You do this via the "Transform > Position > X" controls in the Inspector. We need to shift on the X axis it the amount of empty space to the right. In this case, the full width is 3840 pixels, and the video is 1920, and it's centred, so there are 960 pixels on each side.

First, the right side, since it's on top. We need to shift the X value 960 pixels right:

![SCR-20240105-dy7](https://github.com/CommandPost/FCPCafe/assets/3589315/b83eeed2-def5-4cb6-b4c3-c898e6d6332d)

Then, on the left side, we shift it 960 pixels left (so, negative 960):

![SCR-20240105-dyv](https://github.com/CommandPost/FCPCafe/assets/3589315/04d2cfaf-9c56-4c88-937d-9d286770eb1d)

You can then do this on both screens.

If you prefer to edit your left and right screens independently, rather than shifting each set multiple times, put the whole of the left screen into one Compound Clip and the whole of the Right Screen into another compound clip, then create a new project, put the two left/right compound clips on top of each other and do the adjustment once.

Hope that helps.

**Comment by gsaccani** [comment]
David, this is awesome!!!  Exactly what I was after. Thanks you so much.

**Comment by randomeizer** [comment]
Great! Happy editing.
